### PR TITLE
Add python script to our repo

### PR DESCRIPTION
**Pull request by RoriCremer**
The first step of taking this code from the Hail team is just pasting it into our repo

Successful run:
https://job-manager.dsde-prod.broadinstitute.org/jobs/49d62f48-2dee-417c-aa65-411cbe47be17

GvsQuickstartHailIntegration--we remove the whl from the integration test---sure seems like we wont need one going forward!

Another ticket will be made for these next steps:
Likely this will need to end up in our docker image and the WDL that creates the Avro files can make a version of the input for this scripts instead
Next we will want to remove the Tranches calculations and instead of that value passed in as yet another parameter
Phasing and dropping GQ0

### CODE REVIEW

File: scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl [hunk 1]
```diff
@@ -109,7 +106,6 @@ task CreateAndTieOutVds {
         apt-get -qq update
         apt -qq install -y temurin-8-jdk
 
-        pip install ~{hail_wheel}
         export PYSPARK_SUBMIT_ARGS='--driver-memory 16g --executor-memory 16g pyspark-shell'
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl:109]: pip install --upgrade pip
pip install hail

File: scripts/variantstore/wdl/extract/gvs_avros_to_vds.py [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+           truth_sensitivity_indel_threshold=float,
+           reference_genome=reference_genome_type,
+           partitions_per_sample=numeric,
+           intermediate_resume_point=int,
+           skip_final_merge=bool,
+           unphase=bool,
+           )
+
+
+def import_gvs(refs: 'List[List[str]]',
```
Comment (koncheto-broad) [inline scripts/variantstore/wdl/extract/gvs_avros_to_vds.py:28]: Weren't we going to rename this something other than import_gvs at this stage?  It's obviously not necessary, but I thought I remember the team discussing that

File: scripts/variantstore/wdl/extract/hail_gvs_import.py [hunk 3]
```diff
@@ -24,6 +24,7 @@
 
 def import_gvs(argsfn, vds_path, references_path, temp_path):
```
Comment (koncheto-broad) [inline scripts/variantstore/wdl/extract/hail_gvs_import.py:25]: see my comment above about potentially renaming this function from import_gvs to something else.  It is not strictly necessary, but if it were to be done this is the ticket that we were going to do it in, I believe

**Review by koncheto-broad — COMMENTED** [review]
looks good, just a few questions about potential renaming

File: scripts/variantstore/wdl/extract/gvs_avros_to_vds.py [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+      - `vets` -- The `GT` field is parsed as a Hail :class:`.tcall` and renamed to `LGT`
+        to denote its indexing by local alleles. The `alt` field is split by commas and recorded
+        from strings to integers that refer to an index (0 being ref) into the array of all
+        alleles discovered at a locus. The `GQ` and `RGQ` fields are converted to 32-bit integers.
+      - `site_filtering_data` -- The site filters are converted from a comma-delimited string
+        to a `set<str>` value in Hail. This set contains all unique filters applied, and is
+        an empty set for loci with no record in the input site filtering table.
+      - `vqsr_filtering_data` -- The VQSR data records information specific to a
+        (locus, alternate allele) pair. This data is read in as a dictionary in the row
+        scope of the resulting variant table, where ys keof the dictionary are an alternate
```
Comment (gbggrant) [inline scripts/variantstore/wdl/extract/gvs_avros_to_vds.py:71]: ?ys keof?

File: scripts/variantstore/wdl/extract/gvs_avros_to_vds.py [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+        allele string, where the dictionary values are the full records from the input VQSR
+        filtering table, minus the `ref` and `alt` fields.
+      - `vqsr_tranche_data` -- The VQSR tranche data is recorded as an array of records in the
+        globals of the resulting variant data table.
+
+
+    Execution notes
+    ---------------
+    Currently this method executes a three queries per sample block -- one to collect sample IDs,
```
Comment (gbggrant) [inline scripts/variantstore/wdl/extract/gvs_avros_to_vds.py:80]:
```suggestion
    Currently this method executes three queries per sample block -- one to collect sample IDs,
```

File: scripts/variantstore/wdl/extract/gvs_avros_to_vds.py (hunk 4 above)
Comment (mcovarr) [reply scripts/variantstore/wdl/extract/gvs_avros_to_vds.py:71]: This is [in the original](https://github.com/hail-is/hail/pull/12017/files#diff-ca5f5dc1df328033aa52f76078920a655e48f6cdb683333dcec0adb65c137c8cR3118)... guessing this meant to say "keys of" ?

**Review by mcovarr — APPROVED** [review]

**Review by koncheto-broad — APPROVED** [review]
not an expert on this code, but LGTM

**Pull request merged**